assets/src/adaptivity/scripting.ts

```diff
@@ -75,6 +75,11 @@ export const getExpressionStringForValue = (
         } catch (e) {
           actuallyAString = true;
         }
+      } else {
+        const testVal = getValue('foo', testEnv);
```

Comment:
the variable was getting set as "let stage.iframe.css = video {width: 100%;}*{padding:0;margin:0;}"  which evaluates to undefined. So, I am just returning the actual value if this is the case.